charts/sysdig-deploy/README.md

```diff
@@ -229,7 +242,7 @@ nodeAnalyzer:
 
 ## KSPM Collector
 
-For possible configuration values of the kspm-collector, please refer to the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). All agent-specific configuration can be prefixed with `kspmCollector.` to apply them to this chart.
+For possible configuration values of the kspm-collector, please refer to the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). All specific configuration can be prefixed with `kspmCollector.` to apply them to this chart.
```

Comment:
For the configuration values of the `kspm-collector`, see the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). Prefix all the specific configurations with `kspmCollector.` to apply them to the chart.